packages/utils/src/lib/git.spec.ts

```diff
@@ -1,10 +1,18 @@
 import { expect } from 'vitest';
-import { latestHash } from './git';
+import { CommitData, getLatestCommit } from './git';
 
 describe('git', () => {
-  it('should log current hash', async () => {
-    const hash = await latestHash();
-    expect(hash).toHaveLength(40);
-    expect(hash).toMatch(/^[0-9a-f]+$/);
+  it('should log latest commit', async () => {
+    const commit: CommitData | null = await getLatestCommit();
+    expect(commit).not.toBeNull();
+    expect(commit).toHaveProperty('hash');
+    expect(commit?.hash).toHaveLength(40);
+    expect(commit?.hash).toMatch(/^[0-9a-f]+$/);
+    expect(commit).toHaveProperty('message');
+    expect(commit?.message).not.toHaveLength(0);
+    expect(commit).toHaveProperty('author');
+    expect(commit?.author).not.toHaveLength(0);
+    expect(commit).toHaveProperty('date');
+    expect(commit?.date).not.toHaveLength(0);
```

Comment:
Maybe do a simple format check?

```suggestion
    expect(commit?.date).toMatch(/^\d{4}-\d{2}-\d{2}/);
```